src/events/components/Attendance/AttendButton.tsx

```diff
@@ -12,18 +15,26 @@ const AttendButton: FC<IAttendButtonProps> = (props: IAttendButtonProps) => {
   const dispatch = useDispatch();
   const { eventId } = props;
   const [showModal, setShowModal] = useState<boolean>(false);
+  const [addToast] = useToast({ type: 'success', duration: 5000 });
 
-  const signUp = (token: string | null) => {
+  const signUp = async (token: string | null) => {
     if (token) {
-      dispatch(setAttendeeByEventId({ eventId, captcha: token }));
+      try {
+        await dispatch(setAttendeeByEventId({ eventId, captcha: token })).then(unwrapResult);
+        addToast('Du har blitt meldt på arrangementet');
+      } catch (err) {
+        addToast(`Noe gikk galt under påmeldelse av arrangement, ERROR: ${err.message}`, { type: 'error' });
```

Comment:
Since you are already using `.then()`, could perhaps the `try-catch` be minified to `.then().catch()`? And if you are using `.then()`, could we remove the `await` keyword here by chaining everything with `.then()`?

Reply:
see other answer